test/src/unittests/rhythm/test_percivalenhanceharmonics.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+
+from numpy import *
+from essentia_test import *
+from essentia.standard import MonoLoader, PercivalEnhanceHarmonics
+
+class TestPercivalEnhanceHarmonics(TestCase):
+             
+    def testRegression(self):
+        audio = MonoLoader(filename=join(testdata.audio_dir, 'recorded', 'techno_loop.wav'))()        
+
+        calculatedEnhancedHarmonics = PercivalEnhanceHarmonics()(audio)
```

Comment:
Dont use signal., use autocorrlation.
See how use in beat tracker.
Test requires regression on single frame.

Update the documentation.

Reply:
also the NPY file is very big files.5 MB.

Git rebase to remove 5MB (rebase...check)

Reply:
Use synthetic input instead of real audio in regression (reduce NPY file size , or need to have one)